### PR TITLE
Add spellcheck to pre-commit config

### PR DESCRIPTION
**Pull request by MustkimKhatik**
Related to WordPress/openverse#289

adding shellcheck to openverse-catalog repo

### CODE REVIEW

**Review by dhruvkb — CHANGES_REQUESTED** [review]

**Comment by MustkimKhatik** [comment]
Yeah I did that change

**Comment by dhruvkb** [comment]
@MustkimKhatik ShellCheck is now running as expected, and flagging [a number of warnings and errors](https://github.com/WordPress/openverse-catalog/runs/8251392427?check_suite_focus=true#step:7:90). Could you take a look at the ShellCheck output and implement the changes it recommends?

**Comment by MustkimKhatik** [comment]
Yeah, I'll do them!

**Comment by krysal** [comment]
@MustkimKhatik are you still able to take on the the ShellCheck recommendations?

**Comment by MustkimKhatik** [comment]
Yep, I'm on them.

**Comment by MustkimKhatik** [comment]
Hey @krysal, [current in dag-sync.sh](https://github.com/MustkimKhatik/openverse-catalog/blob/patch-1/dag-sync.sh#L46), it was unused variable and to resolve it I used ```echo```. there was a option to ```export``` too. I will appreciate if you tell me what will be more appropriate here to use?

File: dag-sync.sh [hunk 1]
```diff
@@ -16,14 +16,15 @@ cd "$SCRIPT_DIR"
 
 # Get current commit hash
 current=$(git rev-parse HEAD)
+echo "$current"
```
Comment (dhruvkb) [inline dag-sync.sh:19]: Since it's an unused variable, we could just remove it altogether.

**Review by dhruvkb — APPROVED** [review]
Apart from one change, this looks good to me. Thanks @MustkimKhatik!

**Comment by MustkimKhatik** [comment]
> Apart from one change, this looks good to me. Thanks @MustkimKhatik!

Sure, I'll do that change.

**Review by obulat — APPROVED** [review]
This is such a useful addition, it feels like magic! Thank you for your contribution, @MustkimKhatik, and welcome to Openverse!

**Comment by MustkimKhatik** [comment]
> This is such a useful addition, it feels like magic! Thank you for your contribution, @MustkimKhatik, and welcome to Openverse!

You're welcome and that's my pleasure to work with openverse!

**Review by AetherUnbound — APPROVED** [review]
Thank you for the addition, the changes look great!

**Pull request merged**